packages/types-algosdk/index.d.ts

```diff
@@ -109,20 +109,20 @@ export interface CompileOut {
 export interface Transaction {
   // fields copied from
   // https://github.com/algorand/js-algorand-sdk/blob/develop/src/transaction.js#L117
-  from: ParsedAddress;
-  to: ParsedAddress;
+  from: Address;
+  to: Address;
   fee: number;
   amount: number;
   firstRound: number;
   lastRound: number;
-  note: string;
+  note: Uint8Array;
   genesisID: string;
-  genesisHash: string;
-  lease: number;
+  genesisHash: Buffer;
+  lease: Uint8Array;
 
-  closeRemainderTo: ParsedAddress;
-  voteKey: string;
-  selectionKey: string;
+  closeRemainderTo: Address;
+  voteKey: Buffer;
+  selectionKey: Buffer;
```

Comment:
I think we should use `Uint8Array`. As far as I know SDK is using `Uint8Array` everywhere for bytes / buffer.

Reply:
Seems to me this is being set as `buffer`.
![image](https://user-images.githubusercontent.com/33264364/101959830-21e7d080-3c2c-11eb-8234-95a1cfe72b10.png)
[https://github.com/algorand/js-algorand-sdk/blob/develop/src/transaction.js#L123]()

Reply:
You are right. Sorry.